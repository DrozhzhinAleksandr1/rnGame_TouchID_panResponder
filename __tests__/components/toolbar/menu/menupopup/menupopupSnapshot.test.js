import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import {MenuPopUp}  from '../../../../../app/components/toolbar/menu/menupopup/menupopup';
import stackNav from '../../../../../app/redux/reducers/stacknavreducer';

stackNav.closeDrawer = () => {
    return "test";
}

it('create snapshot MenuPopUp ', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<MenuPopUp navigation={stackNav}/>);
    expect(toolbarSnapshot).toMatchSnapshot();
});
