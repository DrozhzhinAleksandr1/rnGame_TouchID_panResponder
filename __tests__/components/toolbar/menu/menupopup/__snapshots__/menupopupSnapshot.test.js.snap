// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot MenuPopUp  1`] = `
<Component
  style={
    Object {
      "backgroundColor": "#1D2E3E",
      "height": "100%",
    }
  }
>
  <Component
    style={
      Object {
        "backgroundColor": "#1D2E3E",
        "height": 40,
        "width": "100%",
        "zIndex": 2,
      }
    }
  />
  <withNavigation(Connect(LoginBtn))
    closeDrawer={[Function]}
  />
  <Connect(TurnBtn) />
  <withNavigation(Connect(ExitBtn))
    closeDrawer={[Function]}
  />
  <Component
    style={
      Object {
        "backgroundColor": "#5A6772",
        "height": 0.25,
        "marginTop": 32,
        "width": "100%",
      }
    }
  />
  <Connect(LanguageBtns) />
</Component>
`;
